js/main.js

```diff
@@ -215,6 +217,15 @@ function showPage(page) {
       $views[i].classList.add('hidden');
     }
   }
+
+  if (data.entries.length !== 0) {
+    $noEntriesMessage.classList.add('hidden');
+  }
+
+  if (page === 'profile') {
+    $profile.innerHTML = '';
```

Comment:
You should avoid the `innerHTML` property, it is a security risk. Instead, use the `empty` method to delete all children of an element